pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel", "scikit-build", "cmake", "ninja"]
```

Comment:
If `cmake` and `ninja` are already expected to be installed (along with a build tree of ITK), I suggest to remove them from the requirements.

Thinking more about it, I suggest we create a pypi package named `libitk` (same name already used in conda) that would:
* be added as a build requirement [instead of (or in addition to) scikit-build]
* raise an exception when imported. The exception message would point to the build instructions: http://itkpythonpackage.readthedocs.io/en/latest/Build_ITK_Module_Python_packages.html

The rational is that:
*  (1) the `pyproject.toml` is useful only when building with the syntax `pip wheel . -w dist` but that syntax is not usable for us (indeed [here](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/928e6b53ca01cb1a9dcc7d74d6cd5a3993d8aa39/scripts/internal/manylinux-build-module-wheels.sh#L39-L48) is how it is build)
* (2) there are now wheels providing development libraries for ITK